resources/community/devhubOnMinishift.md

```diff
@@ -0,0 +1,115 @@
+
+# Setup Docker in localhost 
+
+## Instruction of How to Deploy Devhub on local Minishift:
+
+Which means you will not Have a nodejs version 10 and s2i-caddy image from Redhat in 'openshift' namespace.
+
+### You will need:
+
+-   Minishift
+-   Docker
+-   Access permission to console.pathfinder.gov.bc/openshift namespace.
+
+#### Step 1: Download and config Minishift Ensure that you download and install  [VirtualBox](https://www.virtualbox.org/wiki/Downloads)  before using the embedded drivers. Install before install minishift.
```

Comment:
Maybe we can shorten the title to just `Step 1: Download and Config Minishift` ? Let's be consistent with capialization too. 

The rest is more of a hint than a title so perhaps it'd be better as a note below the title.